deckhouse-controller/cmd/deckhouse-controller/register-go-hooks.go

```diff
@@ -67,6 +67,8 @@ import (
 	_ "github.com/deckhouse/deckhouse/modules/402-ingress-nginx/hooks"
 	_ "github.com/deckhouse/deckhouse/modules/402-ingress-nginx/requirements"
 	_ "github.com/deckhouse/deckhouse/modules/460-log-shipper/hooks"
+	_ "github.com/deckhouse/deckhouse/modules/490-virtualization/hooks"
+	_ "github.com/deckhouse/deckhouse/modules/491-cdi/hooks"
```

Comment:
cdi module name interferes with cni and csi.

Reply:
proposal — virtualization-cdi.

Reply:
done